### ISSUE TITLE
JavaScript mis-behave.

### ISSUE THREAD

**Issue by ctf0** [shagabutdinov/sublime-semicolon#1]
on js files when i run the plugin to add the missing semi-colon nothing happen ,
but on php files the semi-colon gets added as it expected.

am using this package for js https://packagecontrol.io/packages/SublimeLinter-jshint

**Comment by shagabutdinov** [comment]
1. Which command did you used? There are two: "add semicolon" and "add semicolons".
2. Please send the javascript code where error was appeared. I assume that there are some cases when plugin fails to detect proper semicolon position.

**Comment by ctf0** [comment]
am using the `add_semicolon_to_statements` ,for the code try this

```
success: function(data) {
            $('form').hide()
            $('#result').hide().html(data).fadeIn('slow')
        },
```

i get the linter error but the semi-colons don't get added.

**Comment by shagabutdinov** [comment]
I'm using jsl (https://github.com/SublimeLinter/SublimeLinter-js) and it works fine with provided example, seems there is a problem of how plugin understand different linters messages. I'll be able to fix it only at beginning of next month, sorry. 

Meantime you can try to hack it by yourself, I think the problem somewhere in 43-48 line in file "semicolon.py". Can you add "print(error_text)" at line 47 and send me what is printed in console (ctrl+`)?

**Comment by ctf0** [comment]
all i get is `Missing semicolon`.

**Comment by shagabutdinov** [comment]
Fixed with commit 38d6ba5a12e0141019dba618a85fe98567fcd3ea

**Comment by ctf0** [comment]
much appreciated.
